lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -145,8 +145,8 @@ def answer_labels
               error_class:   "Task export error",
               error_message: "The task cannot be exported",
               context: {
-                classification: @classification,
-                annotation: @annotation,
+                classification: @classification.inspect,
```

Comment:
Perhaps just the classification ID? This inspect might include details like `user_ip`.